packages/engine/Source/Core/AxisAlignedBoundingBox.js

```diff
@@ -47,12 +45,10 @@ function AxisAlignedBoundingBox(minimum, maximum, center) {
 
 /**
  * Creates an instance of an AxisAlignedBoundingBox from its corners.
- *
```

Comment:
I'm personally a big fan of this line break creating visual separation between a description and the tags. It looks like it's easy to enforce this with the plugin too (and auto-fixable) with the [`tag-lines` rule](https://github.com/gajus/eslint-plugin-jsdoc/blob/main/docs/rules/tag-lines.md#readme).

```js
"jsdoc/tag-lines": [
  "error",
  "any",
  {
    startLines: 1,
  },
],
```

Optionally it may be nice to set it or [`sort-tags`](https://github.com/gajus/eslint-plugin-jsdoc/blob/main/docs/rules/sort-tags.md#readme) up for separation between specific blocks of tags but that's a bigger change and may need some manual intervention so I'm fine not inlcuding that for now.